bpf/include/bpf_common.h

```diff
@@ -137,4 +160,31 @@ static inline bool handle_kmesh_manage_process(struct kmesh_context *kmesh_ctx)
     return false;
 }
 
-#endif
\ No newline at end of file
+static inline void *kmesh_get_ptr_val(const void *ptr)
+{
+    /*
+        map_in_map -- outer_map:
+        key		value
+        idx1	inner_map_fd1	// point to inner map1
+        idx2	 inner_map_fd2	// point to inner map2
+
+        structA.ptr_member1 = idx1;	// store idx in outer_map
+    */
+    void *inner_map_instance = NULL;
+    __u32 inner_idx = 0;
+    __u64 idx = (__u64)ptr;
```

Comment:
```suggestion
    __u32 idx = (__u32)(uintptr_t)ptr;
```
According to the design, map-in-map is used to store pointer data. The content stored in the pointer is actually the key (uint32 size) recAccording to the design, map-in-map is used to store pointer data. The content stored in the pointer is actually the key (uint32 size) recorded in map-in-map. It is recommended that the pointer content be explicitly converted into u32 to avoid unnecessary troubles.orded in map-in-map. It is recommended that the pointer content be explicitly converted into u32 to avoid unnecessary troubles.

Reply:
Is that possible two different uint64 equals after casted to uint32

Reply:
This problem does not occur. The max_entry of outter_map does not exceed the maximum value of uint32.

Reply:
@nlwcy can u help to confirm idx indicates the address or value of outermap index：
 __u32 idx = (__u32)(uintptr_t)ptr;，

Reply:
value, not address